ddl-scripts/create_tables_postgres.sql

```diff
@@ -29,6 +29,7 @@ CREATE TABLE IF NOT EXISTS snapshot(
   entity_type VARCHAR(255) NOT NULL,
   persistence_id VARCHAR(255) NOT NULL,
   seq_nr BIGINT NOT NULL,
+  db_timestamp timestamp with time zone NOT NULL,
```

Comment:
wouldn't be difficult to alter table for migration

Reply:
we can remove the NOT NULL here, the fallback can be to use the write_timestamp when it's null. It's probably good enough for the migration scenario. Alternatively lazy loading the timestamp from the event_journal if that column is null. That latter might be better if we are also making the config opt-in on the write side.